cmd/tracee-ebpf/main.go

```diff
@@ -153,7 +153,10 @@ func main() {
 			// Check kernel lockdown
 
 			lockdown, err := helpers.Lockdown()
-			if err == nil && lockdown == helpers.CONFIDENTIALITY {
+			if err != nil {
+				return err
```

Comment:
The question is - do we really want to fail tracee if we can't read lockdown status?
I'm always in favor of trying to be opportunistic and continue with the execution, failing later if we really can't continue. We can log the error instead.

Reply:
Since Lockdown() is an external helper, I wouldn't be comfortable continuing with execution.

In this specific case, could tracee continue to work without a lockdown mode? If so, when it gives an error, I would check these two returns:

`return NOVALUE, fmt.Errorf("could not get lockdown mode")`

Continuing if the error contains "could not get lockdown mode".

Reply:
It is possible that the lockdown file doesn't exists (e.g. older kernel) and tracee will be able to continue.

> Continuing if the error contains "could not get lockdown mode".

Yes, that can be an option

Reply:
Make sure to return a specific error type you can check (instead of checking if error contains). Example: https://github.com/aquasecurity/tracee/blob/main/pkg/cgroup/cgroup.go#L72